ui/Model.elm

```diff
@@ -118,11 +118,11 @@ applyFilters model =
         |> List.filter
             (\buildRecord ->
                 (recordStringEquals (.source >> .product) model.productFilter) buildRecord
-                    && (recordMaybeStringEquals (.target >> .version) model.versionFilter) buildRecord
+                    && (recordStringEquals (.target >> .version) model.versionFilter) buildRecord
                     && (recordStringEquals (.target >> .platform) model.platformFilter) buildRecord
-                    && (recordMaybeStringEquals (.target >> .channel) model.channelFilter) buildRecord
```

Comment:
I believe `recordMaybeStringEquals` isn't used anywhere else, so you can remove its code entirely.